### PR TITLE
Fix toggling between labeling/no-labeling.

### PR DESCRIPTION
**Pull request by tibuch**
@dietzc please take a look at the fix of this issue https://github.com/knime-ip/knip/issues/257.

There was some code in it which was never executed. Now the dialog behaviour is as expected.

### CODE REVIEW

File: org.knime.knip.base/src/org/knime/knip/base/nodes/seg/graphcuts/GraphCutNodeFactory.java [hunk 1]
```diff
@@ -384,23 +372,29 @@ protected void addSettingsModels(final List<SettingsModel> settingsModels) {
...
                     m_lambdaSelection.setEnabled(false);
-                } else {
-                    m_pottsWeight.setEnabled(false);
-                    m_sourceValue.setEnabled(false);
-                    m_useMinMax.setEnabled(false);
-                    m_sinkValue.setEnabled(false);
                 }
 
-                return super.configure(inSpecs);
+                return res;
```
Comment (dietzc) [inline org.knime.knip.base/src/org/knime/knip/base/nodes/seg/graphcuts/GraphCutNodeFactory.java]: are you sure that you can remove the super-call here?

File: org.knime.knip.base/src/org/knime/knip/base/nodes/seg/graphcuts/GraphCutNodeFactory.java [hunk 2]
```diff
@@ -384,23 +372,29 @@ protected void addSettingsModels(final List<SettingsModel> settingsModels) {
...
 
                 if (m_secondColIdx == -1) {
                     m_fgLabel.setEnabled(false);
                     m_bgLabel.setEnabled(false);
                     m_lambdaSelection.setEnabled(false);
-                } else {
-                    m_pottsWeight.setEnabled(false);
-                    m_sourceValue.setEnabled(false);
-                    m_useMinMax.setEnabled(false);
-                    m_sinkValue.setEnabled(false);
```
Comment (dietzc) [inline org.knime.knip.base/src/org/knime/knip/base/nodes/seg/graphcuts/GraphCutNodeFactory.java]: where do you disable those settings models?

File: org.knime.knip.base/src/org/knime/knip/base/nodes/seg/graphcuts/GraphCutNodeFactory.java (hunk 1 above)
Comment (tibuch) [inline org.knime.knip.base/src/org/knime/knip/base/nodes/seg/graphcuts/GraphCutNodeFactory.java]: It is called on line 386.

**Comment by dietzc** [comment]
toggling still doesn't work. try to configure the node with only image input. then you insert e.g. a CCA (append) before the GraphCut. It doesn't to anything.

File: org.knime.knip.base/src/org/knime/knip/base/nodes/seg/graphcuts/GraphCutNodeFactory.java (hunk 2 above)
Comment (tibuch) [inline org.knime.knip.base/src/org/knime/knip/base/nodes/seg/graphcuts/GraphCutNodeFactory.java]: Right, Actually enabling of the other components is missing too.

**Comment by tibuch** [comment]
Now it should work.

**Pull request merged**